docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-snippet/icon-snippet.component.less

```diff
@@ -0,0 +1,20 @@
+:host {
+   display: inline-flex;
+    padding: 10px 0;
+    width: 100%;
+
+    uxd-snippet {
+      display:flex;
+        flex: 1;
+    }
+
+    button {
+        display:flex;
+    }
+}
+
+::ng-deep pre[class*="language-"] {
```

Comment:
This should be moved inside `:host` as ng-deep adds no limitations on the style, this means every `pre` tag on the site with a language class will receive this styling, not just the ones in the component.

Reply:
moved inside <code>:host</code>